client/mergediff_test.go

```diff
@@ -1192,6 +1191,13 @@ func (tc verifyContents) Run(t *testing.T, sb integration.Sandbox) {
 		t.Skip("rootless")
 	}
 
+	// FIXME: TestDiffUpperScratch broken on dockerd and seems flaky with containerd.
+	// 	see https://github.com/moby/buildkit/pull/2726#issuecomment-1070978499
+	// 	and https://github.com/moby/buildkit/pull/2725#issuecomment-1066521712
+	if integration.IsTestDockerd() && tc.name == "TestDiffUpperScratch" {
+		t.Skip("TestDiffUpperScratch is temporarily broken on dockerd")
+	}
```

Comment:
skipping this test for now: https://github.com/crazy-max/buildkit/actions/runs/2018233318